test/VDMSServerTests.cc

```diff
@@ -33,21 +33,33 @@
 
 #include "gtest/gtest.h"
 
-#include "Certificates.h"
+#include "AuthEnabledVDMSServer.h"
 #include "ConnServer.h"
 #include "ExceptionComm.h"
+#include "TLS.h"
 #include "VDMSClient.h"
 #include "VDMSServer.h"
 
 #define SERVER_PORT_INTERCHANGE 43444
 #define NUMBER_OF_MESSAGES 20
 
-typedef std::basic_string<uint8_t> BytesBuffer;
+class VDMSServerTests : public testing::Test {
+protected:
+    void SetUp() override
```

Comment:
Different naming convention here?

Reply:
what do you mean? SetUp is a member of `testing::Test` (third-party).